products/pdt/pdtlib/src/context.rs

```diff
@@ -39,7 +39,11 @@ impl Context {
     }
     pub async fn new(bucket_name: &str, network_name: &str, target_path: &str) -> Result<Self> {
         let region_provider = RegionProviderChain::first_try(Region::new("us-west-2")); // FIXME
-        let config = aws_config::from_env().region(region_provider).load().await;
+        let config = aws_config::from_env()
+            .no_credentials()
```

Comment:
New version of the `aws-rust-sdk` includes the option of skipping authentication. This was causing issues when calling s3 buckets from k8s in staging, previously when using the hacky approach of `make_unsigned`